recipes/reprimand/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+
+export BOOST_ROOT=$PREFIX
+export MBUILD_DIR=build
+
+mkdir -p $MBUILD_DIR
+meson setup --prefix=$PREFIX --libdir=lib $MBUILD_DIR
+cd $MBUILD_DIR
+ninja 
+ninja install
```

Comment:
Is there a `make check` or test we can run there?

Reply:
There are unit tests, but they are run via the meson build system, not make. I was unable to figure out if meson is available in the testing CI environment. If using meson for tests is fine, I would also have to activate building the tests via another option to the meson setup line above.

Reply:
I used "make check" in a generic way. Yes please, try to run the meson tests b/c they are more robust than the "if exist" test below.

Reply:
Ok. Followup question: is it fine to run the test in the build script or do I need to run them in the test section of meta.yaml? I know how to do the former, not sure about the latter.

Reply:
For this kind of this the build script is the right place. Any "post install" tests should be in the meta.yaml or a `run_test.<ext>`.